backend/src/apiserver/template/template_test.go

```diff
@@ -322,3 +324,92 @@ func TestToCrdPeriodicSchedule_NilEndTime(t *testing.T) {
 		StartTime:      &startTime,
 	})
 }
+
+func TestScheduledWorkflow_V1(t *testing.T) {
+	v1Template, _ := New([]byte(v2SpecHelloWorldYAML))
```

Comment:
Shouldn't use a v2 spec for v1 methods.

Reply:
The v2 and v2 here refer to different things. The v2 template refers to the IR version, but this method tests v1 api.